packages/api-client/lib/openapi/apis/doors-api.ts

```diff
@@ -29,13 +29,13 @@ export const DoorsApiAxiosParamCreator = function (configuration?: Configuration
      *
      * @summary  Post Door Request
      * @param {DoorRequest} body
-     * @param {string} doorName
+     * @param {string} door_name
```

Comment:
We should use the same casing as the original ros models. The openapi generator can transform it to `camelCase` but it does not transform the code. i.e. if we send snake_case, the api will error out because the fields are wrong, if we send camelCase, the server will error out because because the fields are wrong.